DuckDuckGo/PrivacyProtectionFooterController.swift

```diff
@@ -61,6 +61,10 @@ class PrivacyProtectionFooterController: UIViewController {
 
 class TrackerNetworkLeaderboardView: UIView {
 
+    struct Constants {
+        static let minimumDetectionLevel = 0.01
```

Comment:
When I specify more, e.g. 10 or 50 I don't see anything being out-filtered, still seeing entries like 4% Or do I misunderstand this functionality?